### DISCUSSION TITLE
All-in-One Solution using Docker - Panel and Single Wings instance in a single package

### DISCUSSION THREAD

**Discussion by bleuthoot-sven** [pelican-dev/panel#461] [Ideas]
Personally I liked to use the previous Pterodactyl panel for hosting personal servers. I've always had issues with setting up a server within using docker. It was possible, but required quite some work to set up.

What I am suggesting is having a docker image and/or docker compose file available that allows for settings up the Panel and a single Wings server manager. Kind of similar of what Nextcloud has with their All-in-One solution. Having this would be very useful setting up Pelican   when having private/homelab servers

**Comment by Boy132** [comment]
There will be a separate docker image/ compose for the panel and wings. A combined image is not planned.
